src/Microsoft.SqlTools.ServiceLayer/TableDesigner/TableDesignerService.cs

```diff
@@ -82,6 +83,16 @@ private async Task HandleProcessTableDesignerEditRequest(ProcessTableDesignerEdi
                            try
                            {
                                // TODO
+                               switch (requestParams.TableChangeInfo.Type)
+                               {
+                                   case DesignerEditType.Add:
+                                       this.HandleAddItemRequest(requestParams);
+                                       break;
+                                   case DesignerEditType.Remove:
```

Comment:
I'm assuming the TODO is about the remove (and maybe others?)

Adding some context there would be helpful

Reply:
I'll be more specific about what to do, all the TODOs here are essential and will be implemented in a few weeks.